src/readii/io/loaders/general.py

```
@@ -0,0 +1,70 @@
+
+    Examples
+    --------
+    >>> config = loadImageDatasetConfig("NSCLC_Radiogenomics", "config/")
+    """
+    # Make full path to config file
+    config_file_path = os.path.join(config_dir_path, f"{dataset_name}.yaml")
+
+    # Check if config file exists
+    if os.path.exists(config_file_path):
+        # Load the config file
+        config = yaml.safe_load(open(config_file_path, "r"))
```

Ruff (0.8.0)

Use a context manager for opening files

(SIM115)